src/libclang/parse_functions.hpp

```diff
@@ -54,6 +54,7 @@ namespace detail
         type_safe::object_ref<const diagnostic_logger> logger;
         type_safe::object_ref<const cpp_entity_index>  idx;
         comment_context                                comments;
+        bool                                           use_c_style;
```

Comment:
Where is this set?

Reply:
It's set based on `compile_config::use_c()` here: https://github.com/foonathan/cppast/blob/5658ef3670edaabcefdf839df8843f2966f860e5/src/libclang/libclang_parser.cpp#L716-L723